tests/sanity/ignore-2.10.txt

```diff
@@ -0,0 +1,2 @@
+plugins/modules/k8s.py validate-modules:parameter-type-not-in-doc
+plugins/modules/k8s.py validate-modules:return-syntax-error
```

Comment:
Are these things we can easily fix or are we doing something special that we can't easily fix? (Or is this coming from something in the upstream module that can't be fixed in this collection anyways?)

Reply:
I'm honestly not sure, saw the sanity tests were failing and pulled these from the community.kubernetes sanity configuration

Reply:
Makes sense. There are a few things that I was never able to resolve in the docs because of the mapping we returned. Those are just bleeding through in the same module here, it seems.